### PR TITLE
added new control node documentation, linked redis documentation

### PR DESCRIPTION
**Pull request by Asparagirl**
added new control node documentation, linked redis documentation

### CODE REVIEW

File: doc/admin.rst [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+
+This pane runs ``spiped`` for Redis, which is used by some but not all pipelines.  ``spiped`` is secure pipe daemon, and it forwards packets from one port to another port.  The preferred connection is ssh tunneling.
+
+Administrators probably won't need to do much in this pane, but it's useful to keep an eye on things.
+
+
+tmux pane 1: pipeline manager
++++++++++++++++++++++++++++++
+
+This pane runs the pipeline manager, which is ```plumbing/updates-listener```.  This listens for updates coming into Redis from all of the many pipelines.  It then sends these updates to a ZeroMQ socket, which is what used by the web-based ArchiveBot dashboard (and possibly a few other things?)l;the dashboard is listening on publicly accessible port 31337.
```
Comment (yipdw) [inline doc/admin.rst]: > things?)l;

Looks like an extra `l` snuck in there?

File: doc/admin.rst [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
+The web-based dashboard has a small unknown memory leak, so the bottom pane runs and monitors ivan's “dashboard killer” daemon. It constantly polls the dashboard to see if it's alive, and it prints a dot if it was a success (dashboard was alive and responded).  If the dashboard does not respond, probably because of that small memory leak, then this daemon kills it and automatically re-spawns it.
+
+
+tmux pane 4: IRC bot
+++++++++++++++++++++
+
+This pane runs the actual ArchiveBot, which is an IRC bot that sits in the channel #archivebot on EFNet and listens for Archive Team volunteers feeding it commands about what websites to archive.
+
+Usually, there's not much that an administrator will need to do for this. If the bot gets kicked off EFnet, it will try to reconnect on its own. However, EFnet sometimes has the tendency to netsplit (disconnect from some IRC noces in a disorganized manner). If that happens, the bot might try to rejoin a server that's been split, in which case the bot might need to be "kicked" (restarted and reconnected to the IRC server).
```
Comment (yipdw) [inline doc/admin.rst]: noces -> nodes

File: doc/admin.rst [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+
+If you need to kick it, hit ``^C`` in this pane to kill the non-responding bot. Then hit the ``Up arrow key`` to show the last command that had been typed into bash, which is usually the one that invokes the bot. You can then adjust that command if you need to (such as possibly changing the server), and then hit enter to re-run that command and reconnect the bot to EFnet.
+
+
+tmux pane 5: redis-cli console
+++++++++++++++++++++++++++++++
+
+This is the console for running redis-cli commands.  It might get closed down, because it's rarely used.
```
Comment (yipdw) [inline doc/admin.rst]: It might be okay to just omit this section -- panes like this pop in and out all the time.

File: doc/admin.rst [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+
+Re-sync the IRC !status command to actual Redis data
+====================================================
+
+The ArchiveBot ``!status`` command that is available in the #archivebot IRC channel on EFnet is supposed to be an accurate counter of how many jobs are currently running, aborted, completed, or pending.  But sometimes it gets un-synchronized from the actual Redis values, especially if a pipeline dies.  Here's how to automatically sync the information again, from Redis to IRC:
+
+	cd /ArchiveBot/bot
+	bundle exec ruby console.rb
+	in_working = $redis.lrange('working', 0, -1); 1
+	in_working.each { |ident| $redis.lren('working', 0, ident) if Job.from_ident(ident, $redis).nil ? }
```
Comment (yipdw) [inline doc/admin.rst]: `lren` -> `lrem`

**Comment by Asparagirl** [comment]
*nydge*

**Comment by Asparagirl** [comment]
er, *nudge*

**Comment by Asparagirl** [comment]
Okay, edits are all done, except I'm Ieaving the notes about tmux pane 5 in (for now).

**Pull request merged**